### ISSUE TITLE
Make build error: downloaders/http.go:25: undefined: grab.GetAsync

### ISSUE THREAD

**Issue by ishuah** [snickers/snickers#169]
Trying to set up the project and i ran into the following error:
```
$ make build
# github.com/snickers/snickers/downloaders
downloaders/http.go:25: undefined: grab.GetAsync
make: *** [build] Error 2
```
Looks like `grab.GetAsync` was [deprecated](https://github.com/cavaliercoder/grab/commit/270162d6afd5ff295dddd38aca5304c3566e3980#diff-4364b09d7b7ecbae6a1623e0755b93fbL72).

The documentation [states](https://github.com/cavaliercoder/grab/blob/master/grab.go#L15) _For non-blocking calls or control over HTTP client headers, redirect policy, and other settings, create a Client instead._

**Comment by thiagozs** [comment]
+1

**Comment by enterprise-easyb** [comment]
Found a work around. I found the commit that removed the GetAsync call. After downloading all of the dependencies, I reset the repo:

cd github.com/cavaliercoder/grab/
git reset --hard  d028be7ddda985b5d10cc060a852cef0cd368ac4

Was able to compile after that.

**Comment by ishuah** [comment]
Nice workaround @enterprise-easyb 
For the the long term snickers should be upgraded to work with the latest version of `github.com/cavaliercoder/grab`.

**Comment by ishuah** [comment]
This issue was fixed by PR #173
